FE/.env.example

```diff
@@ -0,0 +1,8 @@
+API_URL='http://127.0.0.1:8000'
+API_TEST='http://127.0.0.1:8000'
+API_SITE_URL='http://localhost:3000'
```

Comment:
Should be port 3001

Reply:
the variables for the test and live server

Reply:
when I ran it, next started up on 3001